PostHog/PostHogSessionManager.swift

```diff
@@ -22,7 +25,7 @@ class PostHogSessionManager {
             anonymousId = storage.getString(forKey: .anonymousId)
 
             if anonymousId == nil {
-                anonymousId = UUID().uuidString
+                anonymousId = idGen(UUID()).uuidString
```

Comment:
```suggestion
                anonymousId = config.getAnonymousId(UUID()).uuidString
```

Better to use `config.getAnonymousId` directly

Reply:
oh config wasnt cached as a global env., makes sense.